addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -972,5 +983,59 @@ odoo.define('website.tour.form_editor', function (require) {
         },
     ]);
 
+    // Check that the editable form content is actually editable.
+    tour.register("website_form_editable_content", {
+        test: true,
+        url: "/",
+    }, [
+        ...wTourUtils.clickOnEditAndWaitEditMode(),
+        {
+            ...wTourUtils.dragNDrop(snippets[0]),
+            run: "drag_and_drop #wrap",
+        },
+        {
+            content: "Check that a form field is not editable",
+            extra_trigger: ".s_website_form_field",
+            trigger: "section.s_website_form input",
+            run: function () {
+                wTourUtils.fail(this.$anchor[0].isContentEditable, "A form field should not be editable.");
```

Comment:
I am not sure we need an util for that, or at least I don't think using `tour._consume_tour` is right? In general we just do:

```javascript
if (!ok) {
    console.error('...');
}
```
Or is there a specific reason to use `_consume_tour` that I don't see? :thinking: